x/mint/keeper/genesis.go

```diff
@@ -6,8 +6,7 @@ import (
 )
 
 // InitGenesis initializes the x/mint store with data from the genesis state.
-func (k Keeper) InitGenesis(ctx sdk.Context, ak types.AccountKeeper, data *types.GenesisState) {
-	k.SetMinter(ctx, data.Minter)
+func (k Keeper) InitGenesis(ctx sdk.Context, ak types.AccountKeeper) {
```

Comment:
I think we still need to set the minter struct here because it's used directly in the `BeginBlock`

Reply:
+1 tests on this PR fail: https://github.com/celestiaorg/celestia-app/actions/runs/5522450575/jobs/10072054294?pr=2073#step:4:590